src/server/controllers/runs.ts

```diff
@@ -43,6 +43,23 @@ class RunsController {
 
         await ctx.render('index', locals, true);
     }
+
+    async rerunRun (ctx: any, next: any): Promise<void> {
+        debugRuns('Rerunning');
+        const body = ctx.request.body;
+
+        debugRuns(body);
+        if (!('id' in body)) {
+            ctx.status = 400;
+            ctx.body = {
```

Comment:
Usually we add an empty line before and after multiline expressions (conditions, loops, assignments, objects, array, function calls, imports, exports, etc.)